### PR TITLE
add badges with link to docs

### LINKED ISSUE #333: Can't find reference document
I'm assuming there's an autogenerated documentation for the Java, Scala, and Python APIs for the engine.

Where are those? We should have a bad on top pointing to those.

### PR DESCRIPTION
**Pull request by erizocosmico**
Fixes #333 

There's an issue viewing the docs with the `latest` in the redirect. I already opened an issue about it: https://github.com/src-d/docsrv/issues/26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# engine [![Build Status](https://travis-ci.org/src-d/engine.svg?branch=master)](https://travis-ci.org/src-d/engine) [![codecov](https://codecov.io/gh/src-d/engine/branch/master/graph/badge.svg)](https://codecov.io/gh/src-d/engine) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/tech.sourced/engine/badge.svg)](https://maven-badges.herokuapp.com/maven-central/tech.sourced/engine) 
+# engine [![Scala Docs](https://img.shields.io/badge/scala-docs-orange.svg)](https://engine.sourced.tech/latest/api/scala/) [![Python Doc](https://img.shields.io/badge/python-docs-blue.svg)](https://engine.sourced.tech/latest/api/python/) [![Build Status](https://travis-ci.org/src-d/engine.svg?branch=master)](https://travis-ci.org/src-d/engine) [![codecov](https://codecov.io/gh/src-d/engine/branch/master/graph/badge.svg)](https://codecov.io/gh/src-d/engine) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/tech.sourced/engine/badge.svg)](https://maven-badges.herokuapp.com/maven-central/tech.sourced/engine)
```
Comment (campoy) [inline README.md]: are the links correct?
they websites pointed by them look pretty broken and clicking on any link takes to a 404

Comment (ajnavarro) [reply README.md]: This, works well:

Comment (erizocosmico) [reply README.md]: @campoy 

> There's an issue viewing the docs with the latest in the redirect. I already opened an issue about it: src-d/docsrv#26

**Comment by campoy** [comment]
ping?

**Comment by erizocosmico** [comment]
We are waiting for the fix in docsrv
On Wed, 21 Mar 2018 at 19:55, Francesc Campoy <notifications@github.com<mailto:notifications@github.com>> wrote:

ping?

**Comment by ajnavarro** [comment]
@eiso I think you need to rebase with master branch.

**Pull request closed without merging**